features/login/impl/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ plugins {
     alias(libs.plugins.anvil)
     alias(libs.plugins.ksp)
     id("kotlin-parcelize")
```

Comment:
You may add 
`kotlin-parcelize = { id = "org.jetbrains.kotlin.plugin.parcelize", version.ref = "kotlin" }`
to the toml.
Then use:
`alias(libs.plugins.kotlin.parcelize)`